lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -1,12 +1,35 @@
 // @TODO: Move to Forge if this is shipped.
 .counter {
-  display: inline-block;
   color: $med-gray;
   font-weight: $weight-bold;
-  font-size: 13px !important;
-  padding-left: 3px;
+  font-size: 16px;
+  padding-left: 4px;
 
   &.is-active {
-    color: $purple;
+    color: $dark-gray;
+  }
+}
+
+.photo__actions li {
+  * {
+    display: inline;
+    vertical-align: middle;
+  }
+
+  img {
+    width: 30px !important;
```

Comment:
sadly yes, default value it was using is 100%

<img width="241" alt="screen shot 2016-06-09 at 1 51 29 pm" src="https://cloud.githubusercontent.com/assets/897368/15940283/58a3a34c-2e49-11e6-8a40-1c3dfa96f19e.png">

also cant really make it much more specific than this without it looking weird (I think?).